### PR TITLE
Create Page Endpoint/General Pages Endpoints Fixes

### PR DESCRIPTION
**Pull request by PayEstakhri**
- Added Create Page Endpoint
- Fixed payloads for other Pages endpoint calls as well as responses
- Added test for Create Page Endpoint
- Updated tests for other Page API Endpoints
- Updated README with new Create Page Endpoint

### CODE REVIEW

File: src/lib/pages/types.ts [hunk 1]
```diff
@@ -57,14 +57,26 @@ export namespace V2 {
   }
 
   export interface Page {
+    uuid?: string,
```
Comment (jhummel) [inline src/lib/pages/types.ts:60]: I think I'd leave the uuid out of this base type. Then extend it for other types that need it.

File: src/lib/pages/types.ts [hunk 2]
```diff
@@ -57,14 +57,26 @@ export namespace V2 {
...
     title?: string,
     path?: string,
     seo?: Seo
     header_html?: string,
+    draft_status?: 'STAGED_DRAFT' | 'DRAFT'
   }
 
-  export type UpdatePageResponse = void;
-  export type DuplicatePageResponse = void;
+  export interface UpdatePage {
```
Comment (jhummel) [inline src/lib/pages/types.ts:68]: I'd extend the base Page type here:
```
interface ExistingPage extends Page {
  uuid: string
}
```

File: src/lib/pages/types.ts [hunk 3]
```diff
@@ -57,14 +57,26 @@ export namespace V2 {
...
+    uuid?: string,
+    title: string,
+    path: string,
+    seo?: Seo,
+    header_html?: string,
+    draft_status?: 'STAGED_DRAFT' | 'DRAFT'
+  }
+
+  export type UpdatePageResponse = Page;
```
Comment (jhummel) [inline src/lib/pages/types.ts:77]: These next few types will have to change to inherit `ExistingPage` if the above comments are followed.

File: src/lib/pages/types.ts [hunk 4]
```diff
@@ -83,6 +95,11 @@ export namespace V2 {
     page_uuid: string,
   }
 
+  export interface CreatePagePayload {
```
Comment (jhummel) [inline src/lib/pages/types.ts:98]: `page: ExistingPage`

File: src/lib/pages/types.ts (hunk 1 above)
Comment (PayEstakhri) [reply src/lib/pages/types.ts:60]: the uuid is, for some reason, in the schema of every request https://admin-master.dud4.co/admin/api-docs#/Multiscreen%20Sites%20Pages

**Pull request merged**